### ISSUE TITLE
problems autoloading on latex mode

### ISSUE THREAD

**Issue by neripunk** [Kungsgeten/yankpad#90]
Hi, I've been using yankpad for quite a while but I only discovered the possibility to load a category automatically based by matching the category name to a major-mode. It works great for me for org-mode and post-mode (excellent for having email templates available, since I use emacs as an editor for mutt). However, I've been unable to make it work on latex files, when I request a snippet to be inserted, I get the category chooser mini-buffer instead of the snippets corresponding to the mode. I've tried latex-mode, tex-mode, tex-latex-mode, LaTeX/P-mode (the last one is what the mode line lists). If I run describe-mode, I get this regarding the active major modes:

The major mode is LaTeX/P mode defined in tex-mode.el:
This function has :override advice: ‘TeX-latex-mode’.
This is an :override advice, which means that ‘latex-mode’ isn’t run at all, and the documentation below may be irrelevant.

Is this a problem with configuration on my part or a bug? Thanks!

**Comment by Kungsgeten** [comment]
Hi! I don't edit LaTeX that frequently, but I remember seeing LaTeX/P in the mode-line. However when I tried to look into it now I can't find LaTeX/P in tex-mode.el. However I noticed right now, after installing auctex, that the modeline shows LaTeX/P. The symbol `major-mode` still seems to be `latex-mode` though. The function `yankpad-major-mode-category` also seems to returen `"latex-mode"`.

In my configuration it works using `latex-mode` as the "category heading" in the Yankpad file.

I recommend opening your tex-file in a buffer and run `M-: (yankpad-major-mode-category)` and see what it returns.
